plugin/trino-smartnews-functions/src/main/java/io/trino/plugin/smartnews/functions/HashFunctions.java

```diff
@@ -29,15 +30,17 @@
 import java.sql.SQLException;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.Optional;
 
 import static io.trino.spi.type.TypeUtils.readNativeValue;
 import static io.trino.spi.type.VarcharType.VARCHAR;
-import static java.nio.charset.StandardCharsets.UTF_8;
 
 public final class HashFunctions
 {
-    private static final String HASH_KEY = Optional.ofNullable(System.getenv("SN_HASH_KEY")).orElse("smartnews");
+    @SuppressWarnings("checkstyle:StaticVariableName")
+    private static String SN_HASH_KEY = System.getenv("SN_HASH_KEY");
```

Comment:
I am wondering can we just remove SN_HASH_KEY?
Since each query has different queryId, so the input to genSnHashSlice is dynamic already. What do you think?

Reply:
Necessity of SN_HASH_KEY comes from the requirement of [in-reproducibility](https://www.notion.so/Announcement-data-handling-policy-2nd-Party-Designated-PI-8200246de9a44f5a94ab38155a2c6307?pvs=4#bfe2362c68884f83ad248f8e284045b5), and some of discussion is [here](https://www.notion.so/Dynamic-Column-Masking-V2-4e6157cda7074848ba0df79eba928bb7?d=391cc20363274abbb13f6948969800df&pvs=4#447c9f932e5140fe92a64582c3b1692b).
Long story short, I think we can't remove it...